src/components/features/Brands/Brands.js

```diff
@@ -3,35 +3,68 @@ import PropTypes from 'prop-types';
 
 import styles from './Brands.module.scss';
 import Button from '../../common/Button/Button';
-import Brand from '../Brand/Brand';
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
 
-const Brands = ({ brands }) => (
-  <div className={styles.root}>
-    <div className={'container ' + styles.container}>
-      <div className='row'>
-        <div className='col-1 d-flex justify-content-end align-items-center'>
-          <Button variant='small' className={styles.button}>
-            <FontAwesomeIcon icon={faChevronLeft}></FontAwesomeIcon>
-          </Button>
-        </div>
-        {brands.map(brandData => (
-          <Brand key={brandData.id} src={brandData.src} />
-        ))}
-        <div className='col-1 d-flex align-items-center'>
-          <Button variant='small' className={styles.button}>
-            <FontAwesomeIcon icon={faChevronRight}></FontAwesomeIcon>
-          </Button>
+class Brands extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imgToDisplayDesktop1: 6,
+      imgToDisplayDesktop2: 12,
+      activePage: 0,
+    };
+    this.next = this.next.bind(this);
+    this.prev = this.prev.bind(this);
+  }
+  static propTypes = {
+    brands: PropTypes.array,
+  };
+  next(e) {
+    e.preventDefault();
+    if (this.state.activePage === 0) {
+      this.setState({ activePage: this.state.activePage + 1 });
+    }
+  }
+  prev(e) {
+    e.preventDefault();
+    if (this.state.activePage === 1) {
+      this.setState({ activePage: this.state.activePage - 1 });
+    }
+  }
+  render() {
+    const { brands } = this.props;
+    const images1 = brands.slice(0, this.state.imgToDisplayDesktop1);
+    const images2 = brands.slice(6, this.state.imgToDisplayDesktop2);
```

Comment:
A jak dodamy 18 marek to bedzie trzeba zrobić `images3` i poprawić `if-a`? :) Ten kod powinien obsłużyć każdą ilość marek do wyświetlenia ;)

Reply:
Obawiam się, że wszystkie moje dotychczasowe pomysły kończą się koniecznością dodania kolejnego ifa, a już trochę nad tym siedzę :/ mogę prosić o podpowiedź, jak inaczej do tego podejść? :)

Reply:
może spróbuj z własnościwością window.innerWidth

Reply:
Hmm, nie bardzo mi to wychodzi :/ Jak byś tego tutaj użył?

Reply:
const [n, setN] = useState(4);
useEffect(() => {
function changeSlider(){
if(.innerWidth > 1200) wtedy setN(ilosc elementow wyswietlanych)
}
window.addEventListener('resize', changeSlider);
})
, a brands wyswietlaj w taki sposob 
bands .slice(aktywan strona * n, (aktywan strona+ 1) * n) .map

Reply:
Zmiany dodane, ogarnęłam też RWD (czyli task 31 i 32 bym prosiła o sprawdzenie)